fineract-doc/src/docs/en/chapters/architecture/design.adoc

```diff
@@ -40,3 +40,31 @@ NOTE: if a RollbackTransactionAsCommandIsNotApprovedByCheckerException occurs at
 * Process command using handler (In transactional scope).
 * CommandSource object created/updated with all details for logging to 'm_portfolio_command_source' table.
 * In update scenario, we check to see if there where really any changes/updates. If so only JSON for changes is stored in audit log.
+
+= Instance Types Configuring Capabilities to Fineract
```

Comment:
Not sure what this is. The instance type support is already explained: https://fineract.apache.org/docs/current/#_fineract_instance_types

Reply:
cc @edcable.